src/bonds.cpp

```diff
@@ -288,6 +288,48 @@ void PeriodicDihedral::setEnergyFunction(const ParticleVector& particles) {
         const auto angle = std::atan2((norm1.cross(norm2)).dot(v2) / v2.norm(), norm1.dot(norm2));
         return force_constant * (1.0 + std::cos(periodicity * angle - dihedral_angle));
     };
+    forceFunc = [&](Geometry::DistanceFunction dist) -> std::vector<IndexAndForce> {
+        // Torsion on the form a(0) - b(1) - c(2) - d(3)
+        // The need to recalclate the dihedral angle, using the energy function implementation.
+        auto v1 = dist(particles[indices[1]].pos, particles[indices[0]].pos); // a->b
```

Comment:
rename `v1` to `ab`? In CLion, you can use the Refactor menu, or simply shift+F6 to rename a variable.

Reply:
Fixed with https://github.com/mlund/faunus/pull/365/commits/8ff5d87f906a98ff380fb6b34381fbe46ddf24e3.

And thanks for suggestion! 😄